gunicorn/http/wsgi.py

```diff
@@ -344,77 +347,44 @@ def write(self, arg):
         util.write(self.sock, arg, self.chunked)
 
     def can_sendfile(self):
-        return (self.cfg.sendfile and (sendfile is not None))
-
-    def sendfile_all(self, fileno, sockno, offset, nbytes):
-        # Send file in at most 1GB blocks as some operating
-        # systems can have problems with sending files in blocks
-        # over 2GB.
+        return (self.cfg.sendfile and sendfile is not None)
 
-        BLKSIZE = 0x3FFFFFFF
+    def sendfile(self, respiter):
+        if self.cfg.is_ssl or not self.can_sendfile():
+            return False
```

Comment:
Maybe once, on startup, if we can reliably check whether it exists, to warn them that the option is ignored. Here is probably not the place. Maybe we can do that in a separate PR?